From 63792c564c3a96007baf7ce876dea0a6f5f9f880 Mon Sep 17 00:00:00 2001
From: Philip Oakley <philipoakley@iee.email>
Date: Thu, 19 Jan 2023 18:18:25 +0000
Subject: [PATCH 0721/2871] doc: pretty-formats document negative column
 alignments

Commit 066790d7cb0 (pretty.c: support <direction>|(<negative number>) forms,
2016-06-16) added the option for right justified column alignment without
updating the documentation.

Add an explanation of its use of negative column values.

Signed-off-by: Philip Oakley <philipoakley@iee.email>
Signed-off-by: Junio C Hamano <gitster@pobox.com>
---
 Documentation/pretty-formats.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Documentation/pretty-formats.txt b/Documentation/pretty-formats.txt
index 8cc1072196..cbca60a196 100644
--- a/Documentation/pretty-formats.txt
+++ b/Documentation/pretty-formats.txt
@@ -158,7 +158,9 @@ The placeholders are:
 				  Note 2: spaces around the N and M (see below)
 				  values are optional.
 '%<|( <M> )':: make the next placeholder take at least until Mth
-	     display column, padding spaces on the right if necessary
+	     display column, padding spaces on the right if necessary.
+	     Use negative M values for column positions measured
+	     from the right hand edge of the terminal window.
 '%>( <N> )', '%>|( <M> )':: similar to '%<( <N> )', '%<|( <M> )' respectively,
 			but padding spaces on the left
 '%>>( <N> )', '%>>|( <M> )':: similar to '%>( <N> )', '%>|( <M> )'
-- 
2.39.3 (Apple Git-145)

